modules.json

```diff
@@ -406,5 +406,15 @@
       "Clement-Jean"
     ],
     "stars": 0
+  },
+  {
+    "name": "redismodule_qxiu",
+    "license": "MIT",
+    "repository": "https://github.com/fuxingye/redismodule_qxiu",
+    "description": "三个新数据结构, zmap(比原zset多存个value), hnset, looparr, 例如:xlq.zmrevrangebyscore key1 -100 100 withscores withvalues limit 0 100.",
```

Comment:
Please provide a short description in English, with that being the site's only language :)